Sources/Swift/Components/Dates and Timers/IdleTimer/IdleTimer+Gesture.swift

```diff
@@ -15,6 +15,11 @@ extension UIApplication {
     public static var didTimeOutUserInteractionNotification: Notification.Name {
         .init(#function)
     }
+
+    /// - See: `IdleTimer.willTimeoutIdleTimer(duration:for:)`
+    public static var willTimeOutIdleTimerNotification: Notification.Name {
```

Comment:
```suggestion
    public static var willTimeOutUserInteractionNotification: Notification.Name {
```
This makes it consistent with the `didTimeOutUserInteractionNotification` property.